### DISCUSSION TITLE
White screen

### DISCUSSION THREAD

**Discussion by BlackWolfeAku** [lightspark/lightspark#957] [Q&A]
I've installed both lightspeed and palemoon in the hopes of playing some of the saved flash videos and games I've got on my computer. But whenever I try to launch one of them using that browser, all I get is a white screen and it saying done at the bottom. am I missing something?

**Comment by dbluelle** [comment]
Can you provide an example of an URL or swf file that doesn't work?
If you have the swf files on your computer, it may be easier to just start them without the browser, just using the standalone lightspark executable.
